### ISSUE TITLE
Need a way to resize terrain brush

### ISSUE THREAD

**Issue by rkdrnf** [bjorn/tiled#2035]
I'm currently working on a 400 x 400 map.

Drawing tiles one by one (including nearby 8 tiles) with terrain brush is really hard work for making huge area of certain terrain.

If terrain brush supports resize like stamp brush, and apply terrain transformation only to the edge of the brush, it would be far easier to draw huge area!

**Comment by bjorn** [comment]
You're right and this is unfortunately not possible currently. As a workaround, consider drawing the larger areas using lines and filling them with the fill brush, then only using the Terrain Brush to touch up the edges.
